.github/workflows/pull-request.yml

```diff
@@ -36,6 +36,29 @@ jobs:
             
       - name: Run SonarCloud Scanning
         run: |
+          rm '/usr/local/bin/2to3'
```

Comment:
mind adding these in a separate PR rather than here and then rebasing them, please? it will make it easier to revert it later if they go in as a separate change / commit.

Reply:
yep, no probs 🙂

Reply:
Done - #12